components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -292,8 +301,7 @@ public Application updateApplication(ApplicationUpdateRequest updateRequest, Str
                 appDTO.setIdTokenEncryptionMethod(updateRequest.getIdTokenEncryptionMethod());
             }
             if (updateRequest.getRequestObjectSignatureAlgorithm() != null) {
-                appDTO.setRequestObjectSignatureValidationEnabled
```

Comment:
If we are removing this from here are we changing the logic that is using this condition? If we are changing this we will need to change the application management API as well to maintain consistency. 

I think instead of removing this attribute we can set this to true if the algorithm is not none.

Reply:
that is already set.. this was wrongly added here instead of the algorithm